language/de_x_sie/install.php

```diff
@@ -632,3 +259,155 @@
+));
+
+// Updater forms
+$lang = array_merge($lang, array(
+	// Updater types
+	'UPDATE_TYPE'			=> 'Umfang des durchzuführenden Updates',
+
+	'UPDATE_TYPE_ALL'		=> 'Die Dateien und die Datenbank aktualisieren',
+	'UPDATE_TYPE_DB_ONLY'	=> 'Nur die Datenbank aktualisieren',
+
+	// File updater methods
+	'UPDATE_FILE_METHOD_TITLE'		=> 'Methoden zur Datei-Aktualisierung',
+
+	'UPDATE_FILE_METHOD'			=> 'Methode zur Datei-Aktualisierung',
+	'UPDATE_FILE_METHOD_DOWNLOAD'	=> 'Geänderte Dateien in Archiv herunterladen',
+	'UPDATE_FILE_METHOD_FTP'		=> 'Dateien per FTP aktualisieren (automatisch)',
+	'UPDATE_FILE_METHOD_FILESYSTEM'	=> 'Dateien per Direktzugriff aktualisieren (automatisch)',
+
+	// File updater archives
+	'SELECT_DOWNLOAD_FORMAT'	=> 'Archiv-Format für Download auswählen',
+
+	// FTP settings
+	'FTP_SETTINGS'			=> 'FTP-Einstellungen',
+));
+
+// Requirements messages
+$lang = array_merge($lang, array(
+	'UPDATE_FILES_NOT_FOUND'	=> 'Es wurde kein Update-Verzeichnis gefunden. Bitte stellen Sie sicher, dass Sie die richtigen Dateien hochgeladen haben.',
+
+	'NO_UPDATE_FILES_UP_TO_DATE'	=> 'Ihre Version ist auf dem neuesten Stand. Es ist nicht nötig, das Update-Tool auszuführen. Wenn Sie eine Integritätsprüfung der Dateien ausführen möchten, stellen Sie sicher, dass Sie das richtige Update-Paket hochgeladen haben.',
+	'OLD_UPDATE_FILES'				=> 'Die Update-Dateien sind nicht auf dem neuesten Stand. Die gefundenen Update-Dateien sind für ein Update von phpBB %1$s auf phpBB %2$s, aber die neueste Version von phpBB ist %3$s.',
+	'INCOMPATIBLE_UPDATE_FILES'		=> 'Die gefundenen Update-Dateien sind inkompatibel zu Ihrer installierten Version. Ihre phpBB-Version ist %1$s. Das Update-Paket aktualisiert Version %2$s auf %3$s.',
+));
+
+// Update files
+$lang = array_merge($lang, array(
+	'STAGE_UPDATE_FILES'		=> 'Dateien aktualisieren',
+
+	// Check files
+	'UPDATE_CHECK_FILES'	=> 'Prüfe Dateien',
+
+	// Update file differ
+	'FILE_DIFFER_ERROR_FILE_CANNOT_BE_READ'	=> 'Die Datei %s konnte nicht für den Dateivergleich geöffnet werden.',
+
+	'UPDATE_FILE_DIFF'		=> 'Vergleiche geänderte Dateien',
+	'ALL_FILES_DIFFED'		=> 'Alle geänderten Dateien wurden verglichen.',
+
+	// File status
+	'UPDATE_CONTINUE_FILE_UPDATE'	=> 'Dateien aktualisieren',
+
+	'DOWNLOAD'							=> 'Herunterladen',
+	'DOWNLOAD_CONFLICTS'				=> 'Archivdatei mit Konflikten herunterladen',
+	'DOWNLOAD_CONFLICTS_EXPLAIN'		=> 'Suchen Sie nach &lt;&lt;&lt;, um die Konflikte zu finden',
+	'DOWNLOAD_UPDATE_METHOD'			=> 'Archiv mit veränderten Dateien herunterladen',
+	'DOWNLOAD_UPDATE_METHOD_EXPLAIN'	=> 'Nach dem Download sollten Sie das Archiv entpacken. Darin sind die geänderten Dateien enthalten, die Sie in ihr phpBB-Verzeichnis laden müssen. Bitte laden Sie die Dateien in die entsprechenden Verzeichnisse hoch. Nachdem Sie alle Dateien hochgeladen haben, können Sie mit dem Update-Vorgang fortfahren.',
```

Comment:
Auch hier muss an einer Stelle "ihr" groß geschrieben werden und zwar hier: Darin sind die geänderten Dateien enthalten, die Sie in ihr phpBB-Verzeichnis laden müssen.